lms/templates/fields/field_link.underscore

```diff
@@ -1,11 +1,13 @@
-<label class="u-field-title" for="u-field-link-<%- id %>">
+<span class="u-field-title" for="u-field-link-<%- id %>">
     <%- gettext(title) %>
-</label>
+</span>
 <span class="u-field-value">
-    <a id="u-field-link-<%- id %>" href="<%- gettext(linkHref) %>" aria-describedby="u-field-message-<%- id %>">
-        <%- gettext(linkTitle) %>
+    <a id="u-field-link-<%- id %>" href="<%- gettext(linkHref) %>" aria-describedby="u-field-help-message-<%- id %>">
+       <span class="sr">gettext(screenReaderText)</span>
```

Comment:
The bug is fixed. 
What we are trying to do here is, For all the links we have a text that will be used by screenreader, like below.

password: `Reset your Password`
facebook: `Connect your Facebook account`
google: `Connect your Google account`

So we used the below two spans

`<span class="sr"><%- gettext(screenReaderText) %></span>`
`<span aria-hidden="true"><%- gettext(linkTitle) %></span>`

The first span will get appropriate text for `password`, `facebook` and `google` and this text will be used by the screenreader.

The second span will be just the link title, which is `Reset Password` in the case of Password link and `Link` in the case of facebook and google links. We don't want screenreader to use this text so we added the `aria-hidden` attribute.